packages/harpguru-core/src/components/activity-legend/activity-legend.tsx

```diff
@@ -0,0 +1,130 @@
+import { useGlobal } from 'reactn'
+import { useTimingTransition } from 'react-native-redash'
+import Animated, {
+  Easing,
+  interpolate,
+  multiply,
+} from 'react-native-reanimated'
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+import { getPitchIds, getDegreeIds } from 'harpstrata'
+
+import { NoteDisplayFragment } from '../note-display-fragment'
+import { getDisplayValueTuple } from '../../utils'
+import type { DisplayValues } from '../../utils'
+import { DisplayModes } from '../../types'
+import { getSizes, colors } from '../../styles'
+
+const { degreeColors, pageColor, inertOutline } = colors
+
+export const ActivityLegend = (): React.ReactElement => {
+  const [activeHarpStrata] = useGlobal('activeHarpStrata')
+  const [activeDisplayMode] = useGlobal('activeDisplayMode')
+  const {
+    rootPitchId,
+    isActiveComplex: { activePitchIds },
+  } = activeHarpStrata
+  const orderedPitchIds = getPitchIds(rootPitchId)
+  const orderedDegreeIds = getDegreeIds()
+  const sizes = getSizes()
+  const { 9: legendWidth } = sizes
+
+  const activityCells = orderedPitchIds.map((pitchId, index) => {
```

Comment:
Is it misleading to always use the orderedPitchIds as the map source even though we're often displaying the degrees in the end?

Reply:
Raised as tech debt in https://github.com/js-jslog/harpguru/issues/6